Doc/library/operator.rst

```diff
@@ -17,9 +17,18 @@
 
 The :mod:`operator` module exports a set of efficient functions corresponding to
 the intrinsic operators of Python.  For example, ``operator.add(x, y)`` is
-equivalent to the expression ``x+y``.  The function names are those used for
+equivalent to the expression ``x+y``.  Some function names are those used for
 special class methods; variants without leading and trailing ``__`` are also
-provided for convenience.
+provided for backward compatibility.
+
+.. note::
+
+    The dunderless versions of the functions are preferred.
+
```

Comment:
Remote the ``.. note:`` directive and combine the two sentences into one paragraph.  That will get the message across but not litter the visual appearance of the docs with big note blocks that aren't central to actually using the module.   In particular, this note is of less documentation value than the following paragraph that describes the function categories.  It would be better if all of that text (the intro paragraph, the two new sentences, and the categorization paragraph were read as a single block of text with three paragraphs. Breaking it up with *note* blocks impairs intelligiblity and created a weird emphasis on what not to do rather than what to do (see the dev guide suggestions for documenting affirmatively).